### PR TITLE
docs: add section about milestones to contributing

### PR DESCRIPTION
**Pull request by karlbaumhauer**
## Description:
A section about how we work with milestones is added to the contributing guide. Therefore the epic issue template has been deleted.

Closes https://github.com/solid-design-system/internal/issues/38

## Definition of Reviewable:
- [x] Documentation is created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

**Review by yoezlem — APPROVED** [review]

**Review by Vahid1919 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,6 +27,27 @@
...
 - We don't mention the names of internal or external colleagues in issues or other documents hosted on GitHub, as our project is entirely public and can be accessed by anybody. Tagging/mentioning colleagues using their GitHub profiles is fine, as they decided to be visible on GitHub.
 
+### Milestones
+
+We use milestones as a way to group issues and pull requests together towards a specific goal we want to achieve.
+
+Milestones have:
+
+- A name which defines to which goal the milestone is working towards.
+- A start date which defines when the milestone starts.
```
Comment (karlbaumhauer) [inline CONTRIBUTING.md:37]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,6 +27,27 @@
...
+- An end date which defines when the milestone ends.
+- A description which defines what the milestone is about.
```
Comment (karlbaumhauer) [inline CONTRIBUTING.md:39]:
```suggestion
- A description which provides more detailed information about what we want to achieve in this milestone.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,6 +27,27 @@
 - We optimize our components for accessibility.
 - We don't mention the names of internal or external colleagues in issues or other documents hosted on GitHub, as our project is entirely public and can be accessed by anybody. Tagging/mentioning colleagues using their GitHub profiles is fine, as they decided to be visible on GitHub.
 
+### Milestones
+
+We use milestones as a way to group issues and pull requests together towards a specific goal we want to achieve.
+
+Milestones have:
+
+- A name which defines to which goal the milestone is working towards.
```
Comment (karlbaumhauer) [inline CONTRIBUTING.md:36]:
```suggestion
- A name which shortly defines to which goal the milestone is working towards.
```

**Comment by coraliefeil** [comment]
Approved. THX

**Review by van-nguyen-ht — APPROVED** [review]

**Review by DanielHargesheimer — APPROVED** [review]

**Pull request merged**